core/src/main/java/org/jsmart/zerocode/core/report/ZeroCodeReportGeneratorImpl.java

```diff
@@ -93,7 +95,7 @@ public void generateExtentReport() {
         treeReports.forEach(thisReport -> {
 
             thisReport.getResults().forEach(thisScenario -> {
-                ExtentTest test = extentReports.createTest(thisScenario.getScenarioName());
+                ExtentTest test = extentReports.createTest(onlyScenarioName(thisScenario.getScenarioName()));
```

Comment:
- Currently it allows to put the `@@author` anywhere e.g. at the start or end of the scenarioName free text. Also in the middle(but generally users put in the beginning or end of the field.

`onlyScenarioName` will strip everything after the `@@author` mark.

Better, let's keep the original one : `thisScenario.getScenarioName()`

Reply:
The original code added the author name to the author field of the extent report as well as the scenario name. It was redundant information. Also removing the author name from the scenario name made it look cleaner. Anyway, the author name is present in each test which is useful for filtering. 
Maybe we can modify the onlyScenarioName implementation to not strip away anything related to scenarion name except the author name. 

![image](https://user-images.githubusercontent.com/10362257/53300681-af4c0c00-3870-11e9-9783-1f6e7766bb41.png)

Reply:
The current way of displaying author name in scenario name
![image](https://user-images.githubusercontent.com/10362257/53300797-88daa080-3871-11e9-9c6c-74b358eefcae.png)

Reply:
> The current way of displaying author name in scenario name

That's fine for now it helps for exact match for scenario name.

> Maybe we can modify the onlyScenarioName implementation to not strip away anything related to scenarion name except the author name.

Yes, for now, let's not do any logic inside the method. We can implement something later after discussion if anything makes life easy.